docs/faq/index.rst

```diff
@@ -1,17 +1,276 @@
+
+Are AWS IAM roles supports with AWS?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Not currently, our AWS backend just uses ``ACCESS KEY/ ACCESS TOKEN``
+
+Are volume snapshots supported with Flocker?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Not yet.
+
+What happened to ZFS support?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ZFS actually moves data bits around your data center when you need volumes to move; because of this it was inherently slower for the use-cases Flocker is tackling.
+
+Flocker can do everything and more than it did with ZFS with its current supported backends.
+ZFS also has a number of inefficiencies that lead us to move away from support for ZFS and focus on the other backends that we currently support.
+
+We are excited that ZFS was adopted by Canonical/Ubuntu, and we understand some people liked ZFS, but at this moment it is not supported.
+That being said, if we see or hear reasons to support it again, we are always open to having the conversations of why you think we should, so feel free to reach out.
+
+Will you support Nomad, RancherOS, etc?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If we find that enough users want support for Flocker in other frameworks or projects then we will certainly consider it.
+We have looked at adding support to Rancher in the past but it’s not currently something we are working on.
+
+Will you support rkt?
+^^^^^^^^^^^^^^^^^^^^^
+
+Yes, most likely.
+
+If the Open Container Initiative does its part in ensuring a stable API across containers, then we see no reason why we can’t let users plugin and play with other container models.
+
+Security
+--------
+
+I think I've found a security problem, what should I do?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you think you've found a security problem with Flocker (or any other ClusterHQ software), please send a message to security@clusterhq.com.
+Your message will be forwarded to the ClusterHQ security team (a small group of trusted developers) for triage and it will not be publicly readable.
+
+Due to the sensitive nature of security issues, we ask you not to send a message to one of the public mailing lists.
+ClusterHQ has a policy for :ref:`reporting-security-issues` designed to minimize any damage that could be inflicted through public knowledge of a defect while it is still outstanding.
+
+Does Flocker handle security policies?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+At the moment Flocker is configured to use SSL/TLS for its agent and control service communication.
+However, most security policies that have to do with containers are left to the container runtime or orchestration framework.
+Likewise security for volume is managed via the backend that is chosen to run with Flocker.
+Flocker doesn't provide any other container-to-volume based security.
+
+I’m getting an OpenSSL error when I start Flocker services, what should I do?
```

Comment:
This isn't really security, this is just a general operational bug, and belongs more in a troubleshooting section.